lib/modules/manager/jsonnet-bundler/extract.ts

```diff
@@ -46,11 +44,16 @@ function extractDependency(dependency: Dependency): PackageDependency | null {
     return null;
   }
 
-  const match = gitUrl.exec(dependency.source.git.remote);
+  const gitRemote = new URL(dependency.source.git.remote);
```

Comment:
don't use deprecated `url` module, use `parseUrl` from utils

Reply:
Looks like @rarkins started working on the branch, I am not sure if it is ok for me to make changes until this is released, and there are unrelated changes now, maybe a follow up PR afterward?

Reply:
I think it's just a side effect of us rebasing v33. I did "Update Branch" and it seems back to normal for you to continue